gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,137 @@
 ---
-description: Proof Market front end guide
+description: The web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
 
 {% hint style="info" %}
 Please note this product is in beta and expected to change.
 {% endhint %}
 
-#### Proof Market front end can be accessed at [**proof.market**](https://proof.market)
+#### The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
 
 ## New user signup
 
-All users who wish to interact with the proof market must create an account.
+All users who wish to interact with the Proof Market must create an account.
 
 {% hint style="info" %}
-If you do not have the credentials, users can still view the exchange, but they will be unable to post orders to buy/sell proofs.
+If you do not have an account, you can still view the exchange, but you won't be able to post orders
+to buy or sell proofs.
 {% endhint %}
 
-<figure><img src="../.gitbook/assets/image (6).png" alt=""><figcaption></figcaption></figure>
+To register a new user via the web interface, click on **Sign up**:
 
-Click on Sign up
+<figure><img src="../.gitbook/assets/image (6).png" alt=""><figcaption><p>Sign up view</p></figcaption></figure>
```

Comment:
Could you please fix the file names as well? They should have clear names without spaces and other punctuation.

```diff
- image (6).png
+ sign-up.png

- image.png
+ registration-form.png
```

Reply:
fixed